server/lib/csp.js

```diff
@@ -54,10 +59,13 @@ var cspMiddleware = helmet.csp({
     config.get('profile_images_url')
   ]),
   mediaSrc: [BLOB],
-  reportOnly: config.get('csp.reportOnly'),
+  reportOnly: false,
```

Comment:
We can't have two `reportOnly` CSP declarations, we can have one reportOnly and one non.